### PR TITLE
docs/metal: Briefly discuss heterogenous hardware

### PR DESCRIPTION
**Pull request by cgwalters**
Has come up a few times, see also https://github.com/coreos/ignition/issues/1126

### CODE REVIEW

**Review by travier — APPROVED** [review]

**Review by jlebon — COMMENTED** [review]
Hmm, WDYT about making this a subsection or `NOTE` that's under the "Prerequisite" section instead at the top of the page? That's where we talk about producing Ignition configs.

**Comment by cgwalters** [comment]
OK, moved it up.

File: modules/ROOT/pages/bare-metal.adoc [hunk 1]
```diff
@@ -10,6 +10,9 @@ This guide provides instructions to install Fedora CoreOS to bare metal. Three o
 
 Before installing FCOS, you must have an Ignition configuration file and host it somewhere (e.g. using `python3 -m http.server`). If you do not have one, see xref:producing-ign.adoc[Producing an Ignition File].
 
+Note if you have servers with different types and/or number of hard drives, today you must create a separate customized Ignition config for each machine (or machine class).
```
Comment (bgilbert) [inline modules/ROOT/pages/bare-metal.adoc:13]: Let's drop "today".  There are currently no plans to change this.

File: modules/ROOT/pages/bare-metal.adoc [hunk 2]
```diff
@@ -10,6 +10,9 @@ This guide provides instructions to install Fedora CoreOS to bare metal. Three o
...
+A good model is to have the common parts of the configuration factored out into a separate Ignition configuration which can be included (via HTTP or inline) in the per-machine custom config.
```
Comment (bgilbert) [inline modules/ROOT/pages/bare-metal.adoc:14]: To use consistent terminology, "which can be merged (...) into".

**Comment by cgwalters** [comment]
OK addressed those comments.

**Review by bgilbert — APPROVED** [review]
LGTM

**Review by miabbott — APPROVED** [review]
LGTM

**Pull request merged**